H7A3Zi_AzureRTOS_ThreadX/Core/Src/app_threadx.c

```diff
@@ -66,7 +78,11 @@ UINT App_ThreadX_Init(VOID *memory_ptr)
   /* USER CODE END App_ThreadX_MEM_POOL */
 
   /* USER CODE BEGIN App_ThreadX_Init */
-
+  status = tx_semaphore_create(&semaphore, "semaphore-1", 1);
+  if(status == TX_SUCCESS){
+	  tx_thread_create(&thread_ptr1, "thread-1", (void*)thread1_entry_func, 0x0000, thread_stack1, THREAD_STACK_SIZE, 15, 15, 1, TX_AUTO_START);
```

Comment:
You should have used Threadx byte pool allocate api's instead of using thread_stack1 as array arguments.

Reply:
Done